scripts/create_bridges.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service to create bridges
+Before=libvirtd.service
+After=network.target
```

Comment:
A ``Wants=`` or ``Requires=`` doesn't cause create_bridges to fail even if ``network.target``/``network-onine.target`` is disabled.

If we really want  ``create_bridges.service`` to fail if network is not up, then we could do a ``BindsTo=network-online.target``. 

I don't know if we need a strict dependency or if there are any side effects due to it. Services like ``libvirtd.service`` have a simple ``After=network.target`` and it seems to work just fine. 

@henn @SahilTikale @shwsun Thoughts?